tests/test_tracking.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import shutil
+import unittest
+from pathlib import Path
+
+# We use TF to parse the logs
+import tensorflow as tf
+from accelerate import Accelerator
+from tensorboard.plugins.hparams import plugin_data_pb2
+from tensorflow.core.util import event_pb2
+from tensorflow.python.summary.summary_iterator import summary_iterator
+
+
+logger = logging.getLogger(__name__)
+
+
+class TensorBoardTrackingTest(unittest.TestCase):
+    @classmethod
+    def teardown_class(cls):
+        shutil.rmtree("test_project_with_config")
+        shutil.rmtree("test_project_with_log")
+
+    def test_init_trackers(self):
+        hps = None
+        project_name = "test_project_with_config"
```

Comment:
It would probably be best to use a temporary directory here, to avoid users running the tests locally getting this folder.